libs/sdk-ui-pivot/src/impl/cell/cellClass.ts

```diff
@@ -61,16 +64,29 @@ export function cellClassFactory(
         const rowSeparator = !hiddenCell && table.getGroupingProvider().isGroupBoundary(rowIndex);
         const subtotalStyle = row?.subtotalStyle;
 
+        // All new totals combinations
+        const colAndRowSubtotal = subtotalStyle && isColumnSubtotal;
+        const rowTotalAndColSubtotal = isRowTotal && isColumnSubtotal;
+        const rowAndColumnTotal = isRowTotal && isColumnTotal;
+        const rowSubtotalColumnTotal = isRowSubtotal && isColumnTotal;
+
         return cx(
             classList,
             measureIndex !== undefined ? `gd-column-measure-${measureIndex}` : null,
             getCellClassNames(rowIndex, colIndex, hasDrillableHeader),
             `gd-column-index-${colIndex}`,
             isRowTotal ? "gd-row-total" : null,
+            isColumnTotal ? "gd-column-total" : null,
+            isColumnSubtotal ? "gd-column-subtotal" : null,
             subtotalStyle ? `gd-table-row-subtotal gd-table-row-subtotal-${subtotalStyle}` : null,
             hiddenCell ? "gd-cell-hide s-gd-cell-hide" : null,
             rowSeparator ? "gd-table-row-separator s-gd-table-row-separator" : null,
             isTopPinned && isEmptyCell ? "gd-hidden-sticky-column" : null,
+            // All new totals combinations
+            colAndRowSubtotal ? "gd-table-row-column-subtotal" : null,
```

Comment:
I see it's not consistent in other fields as well, but what about having at least these all starting with `is`, like `isColAndRowSubtotal` to be aligned with for example the above `isColumnTotal`

Reply:
Renamed them.